discord.bat

```diff
@@ -1,3 +1,34 @@
+@echo off
+chcp 65001 >nul
+:: 65001 - UTF-8
+
+:: Path check
+set scriptPath=%~dp0
+set "path_no_spaces=%scriptPath: =%"
+if not "%scriptPath%"=="%path_no_spaces%" (
+    echo Путь содержит пробелы. 
+    echo Пожалуйста, переместите скрипт в директорию без пробелов.
+    pause
+    exit /b
+)
+
+:: Cyrillic check
+echo %scriptPath% | findstr /r "[А-Яа-яЁё]" >nul
+if %errorLevel% equ 0 (
+    echo Путь содержит кирилицу. Пожалуйста, переместите скрипт в директорию без кириллических символов.
+    echo Кириллица - Русский алфавит.
+    pause
+    exit /b
+)
+
+:: Admin rights check

```

Comment:
For not-services .bat files admin rights actually not needed, optional for some cases. Same for discord_youtube.bat